File: aifupan_project/ReviewAnalysis/vo/video/DataDiagnosisConfigVo.cs
namespace ReviewAnalysis.vo.video
{
    /// <summary>
    /// 数据诊断配置信息项
    /// </summary>
    public class DataDiagnosisConfigVo
    {
        /// <summary>
        /// 基础设置
        /// </summary>
        public BasicSettingsVo basicSettingsVo { get; set; }

        /// <summary>
        /// 行业 id
        /// </summary>
        public string tradeId { get; set; }

        /// <summary>
        /// 模型 id
        /// </summary>
        public string modelId { get; set; }

        /// <summary>
        /// 模型名称
        /// </summary>
        public string modelName { get; set; }

        /// <summary>
        /// 是否有数据截图 0 没有，1 有
        /// </summary>
        public int? hasDataScreenshot { get; set; }

        /// <summary>
        /// 是否有数据看版 0 没有，1 有
        /// </summary>
        public int? hasBoard { get; set; }
        
        /// <summary>
        /// 是否选择数据截图 0 没有，1 有
        /// </summary>
        public int? selectDataScreenshot { get; set; }

        /// <summary>
        /// 是否选择数据看版 0 没有，1 有
        /// </summary>
        public int? selectBoard { get; set; }

        /// <summary>
        /// 问题 id
        /// </summary>
        public string cueWordsId { get; set; }
        
        /// <summary>
        /// 新问题 id
        /// </summary>
        public string newCueWordsId { get; set; }

        /// <summary>
        /// 生成状态 null: 没有生成过，0 准备开始，1 进行中，2 已完成，3 失败
        /// </summary>
        public int? qaStatus { get; set; }

        /// <summary>
        /// 错误内容
        /// </summary>
        public string errorContent { get; set; }

        /// <summary>
        /// 来源 id
        /// </summary>
        public string sourceId { get; set; }

        /// <summary>
        /// 来源类型 0：视频，1：文件，
        /// </summary>
        public int? sourceType { get; set; }
    }
}

File: aifupan_project/fupan-server/replay-ai/src/main/java/com/jiuyu/replay/ai/vo/DataDiagnosisConfigVo.java
package com.jiuyu.replay.ai.vo;

import com.jiuyu.replay.generic.vo.words.BasicSettingsVo;
import io.swagger.v3.oas.annotations.media.Schema;
import jakarta.validation.constraints.NotNull;
import lombok.Data;

import java.io.Serializable;

/**
 * @author ：lujie
 * @description：
 * @date ：2026/1/19 15:39
 */
@Data
@Schema(description = "数据诊断配置信息项")
public class DataDiagnosisConfigVo implements Serializable {
    private static final long serialVersionUID = 1L;

    @Schema(description = "基础设置")
    private BasicSettingsVo basicSettingsVo;

    @Schema(description = "行业id")
    private Long tradeId;

    @Schema(description = "模型id")
    private Long modelId;

    @Schema(description = "模型名称")
    private String modelName;

    @Schema(description = "是否有数据截图 0没有，1有")
    private Integer hasDataScreenshot;

    @Schema(description = "是否有数据看版 0没有，1有")
    private Integer hasBoard;

    @Schema(description = "是否有数据截图 0没有，1有")
    private Integer selectDataScreenshot;

    @Schema(description = "是否有数据看版 0没有，1有")
    private Integer selectBoard;

    @Schema(description = "问题id")
    private Long cueWordsId;

    @Schema(description = "新问题id")
    private Long newCueWordsId;

    @Schema(description = "生成状态 null: 没有生成过， 0准备开始，1进行中，2已完成， 3失败")
    private Integer qaStatus;

    @Schema(description = "错误内容")
    private String errorContent;

    @Schema(description = "来源id")
    @NotNull(message = "来源id不能为空")
    private String sourceId;

    @Schema(description = "来源类型 0：视频，1：文件，")
    @NotNull(message = "来源类型不能为空")
    private Integer sourceType;

    @Schema(description = "平台类型 0：抖音 1：快手 2：视频号")
    private Integer platform;
}
